source/cost-optimization.txt

```diff
@@ -0,0 +1,22 @@
+=================
```

Comment:
Nit/Question: For drawer pages, did you intentionally leave out the ref at the top?

Reply:
I did, I want these pages to remain just as drawers/don't want people linking to them, but we have to put some content on them for IA reasons